src/MCRun.jl

```diff
@@ -76,14 +96,14 @@ for given ensemble; implemented:
 Asymmetric Metropolis criterium, p = 1.0 if new configuration more stable, 
 Boltzmann probability otherwise
 """
-function metropolis_condition(::NVT, delta_en, beta)
-    prob_val = exp(-delta_en*beta)
+function metropolis_condition(::NVT, d_en, beta)
+    prob_val = exp(-d_en*beta)
     T = typeof(prob_val)
     return ifelse(prob_val > 1, T(1), prob_val)
 end
```

Comment:
renamed delta_en to avoid global variable names

Reply:
Again: I like delta_en or delta_energy more as code easier to read, and I don't think that it is an issue
I am also happy to have it more specific to distinguish between different kinds of delta_en if needed